scripts/asm-installer/install_asm

```diff
@@ -215,6 +225,10 @@ EOF
     return 0
   fi
 
+  if [[ "${VM_EXPANSION}" -eq 1 ]]; then
```

Comment:
Please gate this on enable-gcp-components and otherwise fail if they're not installed

Reply:
In fact, why don't you move this to the block above with init_meshconfig, init_gcp_cas, etc. and then it will be taken care of

Reply:
This is not enabling a GCP component. It is just registering a CR.